File: rnd_number/C#/rnd.cs
// Compile in Visual Studio or using mcs rnd.cs and then running the exe file
// Generates and prints a random number from 0 to 1000

using System;

namespace RandomNumber{

    class Program{

        static void Main(string[] args){

        var rnd = new Random();
        int rndNr = rnd.Next(0,1001); // The higher value is exclusive, putting 1000 will result in 999
        Console.WriteLine("Your random number is: " + rndNr);

        }
    }
}

File: rnd_number/Fortran/rnd.f90
!Compile with gfortran rnd.f90 and run with ./a.out
!Prints a real number as random between 1-1000

program RndNr

implicit none

    real :: r(1)
    write (*,'("Here is a real (pseudo) random number:")')
    call random_number(r)
    print *, floor(r * 1000)

end program
